spk/shairport-sync/src/installer.sh

```diff
@@ -59,6 +61,11 @@ preupgrade ()
 {
     # Stop the package
     ${SSS} stop > /dev/null
+    
+    # Remove firewall config
```

Comment:
I don't know that there's a reason to do this here? It should be sufficient to add the rules on `postinst` and remove them during `preuninst`.